common/src/main/java/io/netty/util/internal/svm/PlatformDependentSubstitution.java

```diff
@@ -0,0 +1,43 @@
+/*
+* Copyright 2019 The Netty Project
+*
+* The Netty Project licenses this file to you under the Apache License,
+* version 2.0 (the "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at:
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations
+* under the License.
+*/
+package io.netty.util.internal.svm;
+
+import com.oracle.svm.core.annotate.Alias;
+import com.oracle.svm.core.annotate.RecomputeFieldValue;
+import com.oracle.svm.core.annotate.TargetClass;
+
+@TargetClass(className = "io.netty.util.internal.PlatformDependent")
+final class PlatformDependentSubstitution {
+    /**
+     * Private constructor to hide this class from netty users.
+     * The constructor is errelevant as it is a SVM substitution.
```

Comment:
Ditto - I guess perhaps we do not need this comment at all because a dev with SVM experience will know it anyway.